src/index.html

```diff
@@ -15,8 +15,117 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <picture class="logo-container">
+        <a
+          href="/"
+          class="logoimage"
+        >
+          <img
+            src="./images/logo.png"
+            alt="MOYO Logo"
+          />
+        </a>
+      </picture>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li>
+            <div id="apple">
+              <a
+                href="#apple"
+                class="menu-item is-active"
```

Comment:
The 'is-active' class is used here, which suggests that this menu item is currently active. Ensure that this class is dynamically managed if the active state changes based on user interaction or page context.